chains/substrate/types.go

```diff
@@ -97,7 +101,10 @@ func (w *writer) createFungibleProposal(m msg.Message) (*proposal, error) {
 
 func (w *writer) createNonFungibleProposal(m msg.Message) (*proposal, error) {
 	tokenId := types.NewU256(*big.NewInt(0).SetBytes(m.Payload[0].([]byte)))
-	recipient := types.NewAccountID(m.Payload[1].([]byte))
+	recipient, err := types.NewAccountID(m.Payload[1].([]byte))
+	if err != nil {
```

Comment:
Is it an API change?

Reply:
yes, it's an constructor interface change